src/heretic/model.py

```diff
@@ -78,27 +108,88 @@ def __init__(self, settings: Settings):
                 # A test run can reveal dtype-related problems such as the infamous
                 # "RuntimeError: probability tensor contains either `inf`, `nan` or element < 0"
                 # (https://github.com/meta-llama/llama/issues/380).
-                self.generate(["Test"], max_new_tokens=1)
+                # We use a simple forward pass instead of generate() to avoid chat template issues.
```

Comment:
What chat template issues? If the chat template doesn't work for whatever reason, it's game over anyway and we don't need to proceed.